Extensions/Multiplayer/multiplayertools.ts

```diff
@@ -107,7 +107,7 @@ namespace gdjs {
       | 'NOT_ENOUGH_PLAYERS'
       | 'UNKNOWN'
       | null = null;
-    let _lobbyId: string | null = null;
+    export let _lobbyId: string | null = null;
```

Comment:
We avoid to export this, instead the getter function is better because it can have logic inside